src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -188,6 +197,11 @@ std::unique_ptr<model_spec> drawing_classifier::init_model() const {
       /* weight_init_fn      */ initializer,
       /* bias_init_fn        */ zero_weight_initializer());
 
+  input_name = std::move(output_name);
```

Comment:
add one more relu.

Reply:
Naming the relu layers independently of the layers they are activating seems weird to me. Could we match the old implementations behavior, which IIRC had layer names like "drawing_dense0" and "drawing_dense0_relu"?

Reply:
yes, sounds reasonable.